package.json

```diff
@@ -2435,7 +2435,7 @@
 				{
 					"command": "review.openFile",
 					"group": "inline@0",
-					"when": "openDiffOnClick && view == prStatus:github && viewItem =~ /filechange(?!:DELETE)/"
+					"when": "openDiffOnClick && view == prStatus:github && viewItem =~ /filechange(?!:DELETE)/ && config.git.showInlineOpenFileAction"
```

Comment:
Thanks for this change! Should this also be added to `pr.openDiffView` to hide that button when configured to `false`?

Reply:
Good reminder, thank you. https://github.com/microsoft/vscode-pull-request-github/pull/6552